
Author: rmh
Status: I hope to be able to merge this (at least in Debian) once Glibc
  defines _SYS_SOUNDCARD_H

--- linux-source-2.6.12/include/linux/soundcard.h	2005-06-17 21:48:29.000000000 +0200
+++ linux-source-2.6.12/include/linux/soundcard.h	2005-08-12 13:38:43.000000000 +0200
@@ -24,6 +22,9 @@
  * SUCH DAMAGE.
  */
 
+#ifndef _SYS_SOUNDCARD_H
+# warning "Never include <linux/soundcard.h> directly; use <sys/soundcard.h> instead."
+#endif
 
 /*
  * OSS interface version. With versions earlier than 3.6 this value is
